src/Utils/FeaturesUtils.js

```diff
@@ -144,8 +147,33 @@ export default {
 
         // We can take this shortcut because either Feature and
         // FeatureCollection have an extent property
-        if (features.extent && !features.extent.isPointInside(coordinate, epsilon)) {
-            return result;
+        if (features.extent) {
+            // Special case, because of the way tiles in VectorTileParser are
+            // handled (see Feature2Texture for a similar solution)
+            if (features.translation || features.scale) {
+                ex.crs = coordinate.crs;
+                features.extent.as(coordinate.crs, ex);
+                if (!ex.isPointInside(coordinate, epsilon)) {
+                    return result;
+                }
+
+                if (!features.translation) {
+                    features.translation = new THREE.Vector3();
```

Comment:
could you avoid new instance?

Reply:
done

Reply:
is useless `features.translation` ans `features.scale` have default values.